omod/src/main/webapp/resources/scripts/metadata/visitTypeService.js

```diff
@@ -0,0 +1,23 @@
+angular.module('visitTypeService', ['ngResource','uicommons.common'])
```

Comment:
Probably this should be moved to uicommons module, see https://issues.openmrs.org/browse/UICM-85

Reply:
Yes, I forgot to delete this from here. I will update the PR.